src/UglyToad.PdfPig.Tests/Geometry/PdfLineTests.cs

```diff
@@ -93,81 +93,5 @@ public void ParallelTo()
             Assert.True(obliqueLine1.ParallelTo(obliqueLine2));
             Assert.True(obliqueLine2.ParallelTo(obliqueLine1));
         }
-
-        [Fact]
-        public void IntersectsWithLine()
-        {
-            var verticalLine1 = new PdfLine(10, 7.5m, 10, 15);
```

Comment:
@BobLd these tests seemed to start failing after changing `PdfLine` and `PdfPoint` to use doubles instead of decimals. I've deleted them for now because I assume it was an accuracy issue.

Reply:
Hi Eliot, no problem I'll fixe them when I have time